server/@types/forms/index.d.ts

```diff
@@ -85,7 +85,7 @@ declare module 'inductionForms' {
     hasWorkedBefore: YesNoValue
   }
 
-  export interface TypeOfWorkExperienceForm {
+  export interface PreviousWorkExperienceForm {
     typeOfWorkExperience: Array<TypeOfWorkExperienceValue>
```

Comment:
The field names within the form object still make sense; its just the enclosing form felt like it had the wrong name